demo/scripts/script-test.js

```diff
@@ -46,8 +47,7 @@ function init() {
       document.querySelector('.notyf').remove();
     } catch (e) {}
   }
-  const code = JSON.parse(document.getElementById('code').value || '{}');
-  notyf = new Notyf(code);
+  notyf = new Notyf(configuration);
```

Comment:
Thanks for the refactor. This is cleaner, indeed, but it fails to fulfill my initial purpose, local testing. When developing, I use to change the configuration code in that textarea and test changes live.

I'm inclined to accept this change and figure out local testing later on my own. I'd appreciate it if you have ideas on how to solve my problem, though.

Reply:
I can re add the textarea and make it invokes `setConfiguration` function to set the configuration with his value

Reply:
Yeah, that sounds good, thanks.